wazuh/wazuh-kibana-app
# Add modal to different sections

Hi team this PR adds a confirmation message when you try to close a form without saving it.
Closes #3183 

### How to test
To try this PR you should follow this steps in this sections: 
- Security: 
  - Users (create / edit)
  - Policies (create / edit)
  - Roles (create / edit)
  - Role Mapping (create / edit)
- Management: 
  - Rules (editor)
  - Decoders (editor)
  - Groups Config (editor)
  - Cluster Condig (editor)


To check that the modal is working you should write something and then try to get out of the editor or flyout. If you haven't change anything the modal shouldn't be displayed

![how-to-test-2021-05-04_17 13 56](https://user-images.githubusercontent.com/46006326/117026910-746d7a80-acfc-11eb-965c-ad10aa132d9b.gif)
